monai/inferers/inferer.py

```diff
@@ -48,7 +46,7 @@ class Inferer(ABC, Generic[T]):
     """
 
     @abstractmethod
-    def __call__(self, inputs: torch.Tensor, network: Callable[..., T], *args: Any, **kwargs: Any) -> T:
+    def __call__(self, inputs: torch.Tensor, network: Callable[..., torch.Tensor], *args: Any, **kwargs: Any) -> Any:
```

Comment:
sometimes the network outputs a list/dict e.g. https://github.com/Project-MONAI/MONAI/blob/385ac0bb6612f88f92e0862a6aee033fb8df5de7/monai/networks/nets/segresnet_ds.py#L405, so could this be
```suggestion
    def __call__(self, inputs: torch.Tensor, network: Callable, *args: Any, **kwargs: Any) -> Any:
```